src/bindings/c/tests/ov_test.hpp

```diff
@@ -51,3 +75,84 @@ inline static std::vector<uint8_t> content_from_file(const char* filename, bool
     }
     return result;
 }
+
+#ifdef OPENVINO_ENABLE_UNICODE_PATH_SUPPORT
+extern const std::vector<std::wstring> test_unicode_postfix_vector;
+inline std::string wstring_to_string(const std::wstring& wstr) {
```

Comment:
We shouldn't duplicate existed functions

Reply:
Maybe you are right, I just want to make the C test code more clear without too much dependent code.

Reply:
done